src/components/Book.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { RemoveBook, FetchBooks } from '../redux/books/booksSlice';
+
+const Book = ({ book, id }) => {
+  const dispatch = useDispatch();
+  const { title, author } = book;
+
```

Comment:
![image](https://github.com/tjay1760/mybookstore/assets/93039359/6fc72e1f-8219-40d9-acc5-55f136ef9e24)


- [ ] Great work so far 👍 but I guess there are some issues left 😨 As you can see in the image above that there is no title and author of the books and the only thing we can see is the Genre 🤔 Kindly fix this issue of displaying book titles and authors to follow the project requirements to make your app work as expected 💯
- [ ] It would be nice if you fix the remove functionality because I am not able to remove even one book from the list which you can see in the image above 😨 Kindly fix it to make your project follow the requirements 🥇

**Hint: You can fix the display books issue by adding [0] after the book at line 8 such as 
`const { title, author } = book[0]`**